src/lib.rs

```diff
@@ -596,8 +768,32 @@ mod tests {
                  0000000000000000000000000000000000000000000000000000000000000000"
             ),
         );
+        assert_eq!(
+            encode_packed(&(
+                7_i32,
+                [
+                    (
+                        address!("0x1111111111111111111111111111111111111111"),
+                        vec![(0x11_i32, 1_i32, 0x12_i32)],
+                    ),
+                    Default::default(),
+                ],
+                vec![
+                    Default::default(),
+                    (
+                        address!("0x0000000000000000000000000000000000001234"),
+                        vec![(0_i32, 0_i32, 0_i32), (0x21, 2, 0x22), (0, 0, 0)]
+                    ),
+                ],
+                8_i32,
+            )),
+            hex!(
+                "00000007111111111111111111111111111111111111111100000011000000010000001200000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000123400000000000000000000000000000021000000020000002200000000000000000000000000000008"
```

Comment:
Can you break up these lines? `rustfmt` has issues with overly long lines that it does not know how to split.